plugin-core/src/main/java/com/salesforce/bazel/eclipse/config/BazelEclipseProjectFactory.java

```diff
@@ -196,20 +196,12 @@ private static void importBazelWorkspacePackagesAsProjects(BazelPackageInfo pack
         if (eclipseProject != null) {
             importedProjectsList.add(eclipseProject);
         }
-
-        for (BazelPackageInfo childPackageInfo : packageInfo.getChildPackageInfos()) {
-            try {
-                importBazelWorkspacePackagesAsProjects(childPackageInfo, bazelWorkspaceRoot, importedProjectsList);
-            } catch (Exception anyE) {
-                anyE.printStackTrace();
-            }
-        }
```

Comment:
I removed this because I don't think we use it, and it simplifies the import logic: the list of bazel packages to import is always a flat list of, conceptually, fully qualified paths to bazel packages.  This is how the UI works today anyway.  And when importing because of projectview file changes, that will also be the case.

Reply:
Cool. On the topic of the flat list, at some point we will want to order it. We want the first projects to be imported to be the upstream deps of the latter imported projects. Not part of project views, but that is on the horizon. That will be the key to speeding up perf of import.